src/Tests/dotnet-workload-install.Tests/GivenWorkloadManifestUpdater.cs

```diff
@@ -345,12 +339,56 @@ public void GivenWorkloadsAreOutOfDateUpdatesAreAdvertisedOnRestoringCommands(st
 
         }
 
+        [Fact]
+        public void TestSideBySideUpdateChecks()
```

Comment:
It would probably be a good idea to add a comment describing what the test basically does, ie checking that different version bands don't interfere with each other's update check timers.